modules/social_features/social_profile/src/Plugin/Field/FieldWidget/SocialSplitTagWidget.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace Drupal\social_profile\Plugin\Field\FieldWidget;
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\select2\Plugin\Field\FieldWidget\Select2EntityReferenceWidget;
+use Drupal\social_profile\SocialProfileTagServiceInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class SocialSplitTagWidget.
+ *
+ * @FieldWidget (
+ *   id = "social_split_tag",
```

Comment:
@dmytriikaiun From the code, I see this widget works only for the profile tag field.
So maybe it's better to rename the machine name and class to something with a Social Profile prefix.